ludwig/datasets/ohsumed/ohsumed.py

```diff
@@ -0,0 +1,74 @@
+import csv
+import pandas as pd
+from typing import Dict
+from ludwig.datasets.base_dataset import BaseDataset
+import shutil
+import os
+
+
+"""A class to process the ohsumed training data"""
+import yaml
+
+
+class OhsuMed(BaseDataset):
+
+    def __init__(self):
+        self.__initial_path = os.path.abspath(os.path.dirname(__file__))
+        self.__source_location =  os.path.join(self.__initial_path , "../text/ohsumed/ohsumed-allcats.csv")
```

Comment:
No these datasets cannot, I did some research on both of these, for every dataset that can be downloaded using a URL, I'll write a simple utility to do this, stay tuned for a future PR on that once we get the basics in place

Reply:
Can you explain the difference between the datasets included here and the ones here: https://www.mat.unical.it/OlexSuite/Datasets/SampleDataSets-download.htm.  If we can obtain them from this link, we should do that instead of including them in the repo.

If for whatever reason we cannot obtain them automatically via a script (e.g., we need the user to authenticate or something), then we should host them ourselves in something like S3 (which I can do using the Horovod S3 account).  If we go this route, we need to be extra certain that the datasets are in the public domain, and not subject to any copyrights preventing us from hosting them ourselves.

In general, I don't think we should host them in the repo itself, however, as datasets can get very large and add a lot of bloat to git pull / clone.

Reply:
@tgaddair point taken let me explore this further, I didnt see this link, will keep you posted

Reply:
The file download capability is added now.